CHANGELOG.md

```diff
@@ -3,6 +3,118 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.32](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.31...v4.0.0-alpha.32)
+
+_June 18, 2021_
+
+Big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚡️ Components that use portals, like `Select` and `Autocomplete`, can now be used in the cell editing (#1772) @m4theushw
+- 📃 Apply the `valueFormatter` to the CSV exporting (#1922) @DanailH
+- 💅 Rename CSS classes to follow the BEM convention (#1872) @DanailH
```

Comment:
I think that this framing will fly better among the audience as they seek consistency.

```suggestion
- 💅 Rename CSS classes to match the convention of the core components (#1872) @DanailH
```